### PR TITLE
Feature/pid-381

### PR DESCRIPTION
**Pull request by Kolezhniuk**

### CODE REVIEW

File: src/credentials/credential-wallet.ts [hunk 1]
```diff
@@ -340,19 +344,16 @@ export class CredentialWallet implements ICredentialWallet {
...
-    } else {
-      cr.credentialStatus = {
-        id: `${hostUrl}/revocation/${request.revNonce}`,
-        revocationNonce: request.revNonce,
-        type: CredentialStatusType.SparseMerkleTreeProof
-      };
-    }
+    const id =
+      request.revocationOpts.type == CredentialStatusType.SparseMerkleTreeProof
+        ? `${request.revocationOpts.baseUrl}/revocation/${request.revocationOpts.nonce}`
```
Comment (vmidyllic) [inline src/credentials/credential-wallet.ts:349]: ? `${request.revocationOpts.baseUrl}/${request.revocationOpts.nonce}` :

File: src/identity/identity-wallet.ts [hunk 2]
```diff
@@ -93,16 +81,12 @@ export interface IIdentityWallet {
...
    * @param {IdentityCreationOptions} opts - default is did:iden3:polygon:mumbai** with generated key.
    * @returns `Promise<{ did: DID; credential: W3CCredential }>` - returns did and Auth BJJ credential
    * @beta
    */
 
-  createIdentity(
-    hostUrl: string,
-    opts?: IdentityCreationOptions
-  ): Promise<{ did: DID; credential: W3CCredential }>;
+  createIdentity(opts?: IdentityCreationOptions): Promise<{ did: DID; credential: W3CCredential }>;
```
Comment (vmidyllic) [inline src/identity/identity-wallet.ts:89]: here it's optional, in the implementation it's not

**Review by vmidyllic — CHANGES_REQUESTED** [review]

**Comment by Kolezhniuk** [comment]
@vmidyllic fixed your comments

File: index.html [hunk 3]
```diff
@@ -56,10 +58,16 @@
...
-            'http://rhs.com/node',
-            seedPhrase
+        const { did, credential } = await wallet.createIdentity({
+            method: core.DidMethod.Iden3,
+            blockchain: core.Blockchain.Polygon,
+            networkId: core.NetworkId.Mumbai,
+            seed: seedPhrase,
+            revocationOpts: {
+                type: CredentialStatusType.Iden3ReverseSparseMerkleTreeProof,
+                baseUrl: 'http://rhs.com/node'
```
Comment (nedgar) [inline index.html:68]: rhs.com is an in-use domain. Suggest using the actual Polygon ID RHS.

**Pull request merged**